Add WildFly and Zookeeper files (#175)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/wildfly.md

---
name: WildFly

> category: Web

Author: ```suggestion
category: Web Server
```

> description: A powerful, modular, & lightweight application server that helps you build amazing applications.

Author: ```suggestion
description: WildFly is a powerful, modular and lightweight application server that helps to build amazing applications.
```

download_url: https://github.com/wildfly/wildfly/tags
works_on_arm: true
supported_minimum_version:
    version_number: 18.0.1
    release_date: 15/11/2019

optional_info:
    homepage_url: https://www.wildfly.org/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:  
        partner_content: 
        official_docs: https://github.com/wildfly/wildfly/blob/main/README.md
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum: 
    release_notes__recommended_minimum: 

>     other_info: There are no release notes or binaries present for Linux/ARM64. Wildfly 18.0.1 is successfully installed and tested on the Neoverse N1, following the steps mentioned in README.md[https://github.com/wildfly/wildfly/blob/18.0.0.Final/README.md].

Author: ```suggestion
    other_info: There are no release notes or binaries present for Linux/ARM64. Wildfly 18.0.1 is successfully installed and tested on the Neoverse N1, following the steps mentioned in [README.md](https://github.com/wildfly/wildfly/blob/18.0.0.Final/README.md).
```
